patches/api/0457-Add-ShulkerDuplicateEvent.patch

```diff
@@ -36,7 +36,8 @@ index 0000000000000000000000000000000000000000..eb32542031088b1735fd0cc90fe350d0
 +    }
 +
 +    /**
-+     * Gets the shulker that is duplicating.
++     * Gets the shulker that is duplicating. The new entity that is being created,
++     * and that does not exist in the world yet.
 +     *
 +     * @return The Child
```

Comment:
(maybe should write "child" with a small letter, similar to "parent" in the corresponding method just below?)